5_databases/week3/week3.sql

```diff
@@ -0,0 +1,123 @@
+USE week3_1;
+
+-- Get all meals
+SELECT *
+from meal;
+
+-- Add a new meal
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) VALUES 
+('Elephant brain', 'it will be a starter', 'North Korea', '2024-10-25 06:54:16', 8, 200.50, '2023-09-10 11:05:05');
+
+-- Get a meal with any id, fx 1
+SELECT *
+from meal
+WHERE id = 1;
+
+-- Update a meal. Update any attribute fx the title or multiple attributes
+UPDATE meal
+SET location = 'Berlin'
+WHERE price >= 200;
```

Comment:
Correct, but you are asked to update a meal _with any id_

Reply:
It says any id, fx 1. So I chose 1 and update it. Is it wrong?